pages/account/manage/profile.js

```diff
@@ -250,10 +243,11 @@ export default function Profile({ BASE_URL, profile, fileExists }) {
                         onTagAdd={handleTagAdd}
                         onTagRemove={handleTagRemove}
                         tags={tags}
-                        inputRef={tagInputRef}
+                        showNotification = {showNotification}
+                        setShowNotification = {setShowNotification}
                       />
                       <p className="text-sm text-primary-medium-low dark:text-primary-low-high">
-                        Separate tags with commas.
+                        Separate tags with commas and Maximum 32 characters allowed in each tag.
```

Comment:
```suggestion
                        Separate tags with commas (max 32 characters).
```

Reply:
should i change this 'Maximum 32 characters allowed in each tag.' to this '(max 32 characters)'

Reply:
I already made the change, it is marked as resolved